model/src/test/kotlin/JsonSchemaTest.kt

```diff
@@ -73,6 +73,19 @@ class JsonSchemaTest : StringSpec() {
 
             errors should beEmpty()
         }
+
+        "package-configuration.yml validates successfully" {
+            val packageConfigurationSchema = JsonSchemaFactory
+                .builder(JsonSchemaFactory.getInstance(SpecVersion.VersionFlag.V4))
```

Comment:
BTW, any idea why we're using V4 here, but V7 for `curations.yml`? I mean, yes these flags correspond to the *draft-XX* in the schema files, but why are we using different version after all?

Reply:
V7 in the `curations-schema.yml` is required because I'm using if-then-else there which is only available for draft-07.
I explained it in this commit in the follow up PR: https://github.com/oss-review-toolkit/ort/pull/5395/commits/cb9da0a2d7d6e7c66253580590485c25ca7d9c62